modules/migration-migrating-applications-api.adoc

```diff
@@ -1,92 +1,35 @@
-= Migrating your applications with the {mtc-full} API
+= Migrating your applications from the command line
 
-You can migrate your applications on the command line with the {mtc-full} ({mtc-short}) API.
-
-You can migrate applications from a local cluster to a remote cluster, from a remote cluster to a local cluster, and between remote clusters.
-
-This procedure describes how to perform indirect migration and direct migration:
-
-* Indirect migration: Images, volumes, and Kubernetes objects are copied from the source cluster to the replication repository and then from the replication repository to the destination cluster.
-* Direct migration: Images or volumes are copied directly from the source cluster to the destination cluster. Direct image migration and direct volume migration have significant performance benefits.
-
-You create the following custom resources (CRs) to perform a migration:
-
-* `MigCluster` CR: Defines a `host`, local, or remote cluster
-+
-The `migration-controller` pod runs on the `host` cluster.
-
-* `Secret` CR: Contains credentials for a remote cluster or storage
-* `MigStorage` CR: Defines a replication repository
-+
-Different storage providers require different parameters in the `MigStorage` CR manifest.
-
-* `MigPlan` CR: Defines a migration plan
-* `MigMigration` CR: Performs a migration defined in an associated `MigPlan`
-+
-You can create multiple `MigMigration` CRs for a single `MigPlan` CR for the following purposes:
-+
-* To perform stage migrations, which copy most of the data without stopping the application, before running a migration. Stage migrations improve the performance of the migration.
-* To cancel a migration in progress
-* To roll back a completed migration
-
-.Prerequisites
-
-* You must have `cluster-admin` privileges for all clusters.
-* You must install the {product-title} CLI (`oc`).
-* You must install the {mtc-full} Operator on all clusters.
-* You must use the same version of the installed {mtc-full} Operator on all clusters.
-* You must configure an object storage as a replication repository.
-* If you are using direct image migration, you must expose a secure registry route on all remote clusters.
-+
-To create the route, run the following command on a remote cluster:
-
-** For {product-title} 3:
-+
-[source,terminal]
-----
-$ oc create route passthrough --service=docker-registry --port=5000 -n default
-----
-** For {product-title} 4:
-+
-[source,terminal]
-----
-$ oc create route passthrough --service=image-registry --port=5000 -n openshift-image-registry
-----
-
-* If you are using direct volume migration, the source cluster must not have an HTTP proxy configured.
+You can migrate your applications from the command line with the {mtc-full} ({mtc-short}) API.
 
 .Procedure
 
-. Create a `MigCluster` CR manifest for the `host` cluster called `host-cluster.yaml`:
+. Create a `MigCluster` CR manifest for the host cluster:
 +
 [source,yaml]
 ----
+$ cat << EOF | oc apply -f -
```

Comment:
Just an FYI for the future, we recently discussed in an OpenShift docs team mtg that we're going to update the guidelines (WIP currently) to discourage use of these `EOF`-style command structures, since they don't work for Windows users.

Reply:
Does 'oc' work on Windows?

Reply:
Yes: https://docs.openshift.com/container-platform/4.7/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli-on-windows_cli-developer-commands